arwen/elrondapi/elrondei.go

```diff
@@ -1359,9 +1384,13 @@ func int64storageLoad(context unsafe.Pointer, keyOffset int32, keyLength int32)
 
 	data := storage.GetStorage(key)
 
-	bigInt := big.NewInt(0).SetBytes(data)
+	valueBigInt := twos.SetBytes(big.NewInt(0), data)
```

Comment:
is this backward compatible ?

Reply:
Do we need the same in `int64storageStore()`?

Reply:
no :(
will redesign

Reply:
@camil yes, slipped me.
But will rewrite anyway